source/tests/consistent/fitting/test_ener.py

```diff
@@ -42,7 +42,9 @@
     ("float64", "float32", "bfloat16"),  # precision
     (True, False),  # mixed_types
     (0, 1),  # numb_fparam
+    (0, 1),  # numb_aparam
     ([], [-12345.6, None]),  # atom_ener
+    (True, False),  # use_aparam_as_mask
```

Comment:
`numb_aparam == 0 and use_aparam_as_mask == True` can be skipped, which is not a meaningful parameter.